resources/json/PBACDefaults_config.json

```diff
@@ -18,7 +18,7 @@
             "_id": "submission_request:create:all",
             "group": "Submission Request Permissions",
             "name": "Create",
-            "inherited": ["submission_request:view"],
+            "inherited": ["submission_request:view:all"],
```

Comment:
The scope of inherited values should match that of the permission. It applies other permission.